### PR TITLE
Added test endpoints, small refactoring

### PR DESCRIPTION
**Pull request by Mingela**

### CODE REVIEW

File: deploy/docker-compose.yml [hunk 1]
```diff
@@ -78,6 +78,7 @@ services:
       WAIT_HOSTS_TIMEOUT: 60
       CREDENTIAL_PUBKEY: b9679bbf526a1c936cd1144b56a370d376fa8246b248cd72f952b45a2f20bdad
       CREDENTIAL_PRIVKEY: 56a3c52cd039d9b73a1720052600a20962350b1ea169b4783cefbf87ed99406a
+      WHITELIST_VALIDATION: 1
```
Comment (Alexey-N-Chernyshov) [inline deploy/docker-compose.yml]: Please, move confirmation time to docker-compose.dev or docker-compose.ci

File: eth/src/main/kotlin/com/d3/eth/sidechain/util/DeployHelper.kt [hunk 2]
```diff
@@ -29,6 +29,8 @@ import java.math.BigDecimal
 import java.math.BigInteger
 import java.util.concurrent.TimeUnit
 
+const val ENDPOINT_ETHEREUM = "eth"
```
Comment (dolgopolovwork) [inline eth/src/main/kotlin/com/d3/eth/sidechain/util/DeployHelper.kt:32]: Why do we store endpoint data inside the `DeployHelper` file?

Comment (Mingela) [reply eth/src/main/kotlin/com/d3/eth/sidechain/util/DeployHelper.kt:32]: Inside utility tools file, what's contradiction? Where to store it then?

File: deploy/docker-compose.yml (hunk 1 above)
Comment (Mingela) [reply deploy/docker-compose.yml]: Moved to dev

**Review by Alexey-N-Chernyshov — APPROVED** [review]

**Pull request merged**